pkgs/development/python-modules/pyproj/default.nix

```
@@ -28,33 +36,59 @@ buildPythonPackage rec {
     })
   ];
 
-  buildInputs = [ cython pkgs.proj ];
+  nativeBuildInputs = [ cython pkgs.tree ];
+  buildInputs = [ pkgs.proj ];
```

```suggestion
  nativeBuildInputs = [ cython tree ];
  buildInputs = [ proj ];
```

Please inherit from top level instead.

---

Done.